test-data/unit/check-typeddict.test

```diff
@@ -730,7 +730,7 @@ p['x'] = 1
 from mypy_extensions import TypedDict
 TaggedPoint = TypedDict('TaggedPoint', {'type': str, 'x': int, 'y': int})
 p = TaggedPoint(type='2d', x=42, y=1337)
-p['x'] = 'y'  # E: Argument 2 has incompatible type "str"; expected "int"
+p['x'] = 'y'  # E: Argument "x" has incompatible type "str"; expected "int"
```

Comment:
I think the message purposed in the original issue 'Value of "foo" has incompatible type "str"; expected "int"' is more clear since x is not an argument.